### ISSUE TITLE
Request getSession() cause an UndefinedMethodException

### ISSUE THREAD

**Issue by blitzen12** [snc/SncRedisBundle#485]
Calling the getSession() method from request object cause an UndefinedMethodException
Attempted to call an undefined method named "setLogger" of class "Predis\Connection\StreamConnection".

```
public getUserInfoAction(Request $request){
     $session = $request->getSession();
}
```
config.yml
```
snc_redis:
    clients:
        session:
          type: predis
          alias: session
          dsn: redis://password@localhost:6379
    session:
        client: session
        prefix: nexlife_
framework:
    session:
        handler_id: snc_redis.session.handler
```

**Comment by xabbuh** [comment]
Can you show the full stack trace for this exception?

**Comment by blitzen12** [comment]
I cannot find the logs for stack trace so I'll just paste an image link.
https://ibb.co/HTt5QkV

**Comment by B-Galati** [comment]
@blitzen12 Thanks for reporting.
Which version of PHP, SncRedisBundle and Symfony are you using please?

**Comment by xabbuh** [comment]
What is the value of the `snc_redis.connection_wrapper.class` parameter?

**Comment by blitzen12** [comment]
@xabbuh I dont have that configuration in my config file.

**Comment by xabbuh** [comment]
@blitzen12 You can run the following command: `php bin/console debug:container --parameter=snc_redis.connection_wrapper.class`

**Comment by blitzen12** [comment]
@xabbuh below is the output when I execute the command you mention.

  Parameter                            Value
 ------------------------------------ --------------------------------------------
  snc_redis.connection_wrapper.class   Snc\RedisBundle\Client\Predis\Connection\ConnectionWrapper
 ------------------------------------ --------------------------------------------

**Comment by xabbuh** [comment]
I still do not really understand how that could lead to the aforementioned error, but can you please check nonetheless if #489 makes a difference?

**Comment by xabbuh** [comment]
should be fixed by #489

**Comment by B-Galati** [comment]
Just released 2.1.8 with the mentioned fix.
Feel to reopen the issue if something is wrong.
